climt/_lib/rrtmg_sw/rrtmg_sw_rad.nomcica.f90

```diff
@@ -808,8 +808,8 @@ subroutine rrtmg_sw &
             swhrc(iplon,i) = (swnflxc(i+1) - swnflxc(i)) * zdpgcp
             swhr(iplon,i) = (swnflx(i+1) - swnflx(i)) * zdpgcp
          enddo
-         swhrc(iplon,nlayers) = 0._rb
-         swhr(iplon,nlayers) = 0._rb
+         !swhrc(iplon,nlayers) = 0._rb
```

Comment:
Remove commented code.

Reply:
I want to keep this, since this is the original code. I don't know why they have done this, but
it makes RRTMG non-conservative.

Reply:
If you're going to do that, you should add a comment as to why it's commented out.